src/gbo/GameBoyCore.js

```diff
@@ -7895,12 +7895,20 @@ GameBoyCore.prototype.memoryReadJumpCompile = function () {
 						return 0x8F | parentObj.memory[0xFF75];
 					}
 					break;
-				case 0xFF76:
-				case 0xFF77:
-					this.memoryHighReader[index & 0xFF] = this.memoryReader[index] = function (parentObj, address) {
-						return 0;
-					}
-					break;
+                case 0xFF76:
+                    //Undocumented realtime PCM amplitude readback:
```

Comment:
The whole GB/GBC emulator core is a shitshow that should be discontinued for a rewritten one. There's a lot of not obvious structural timing issues that requires a lot of thought put in to track down. @_@